### PR TITLE
TFKUBE-234: Added the message in case of unsuccessful attempt resuming the server

### PR DESCRIPTION
**Pull request by nghazali**

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -230,7 +231,14 @@ resume_bamboo_server() {
...
       bamboo_url=$(terraform output | grep '"bamboo" =' | sed -nE 's/^.*"(.*)".*$/\1/p')
       resume_bamboo_url="${bamboo_url}/rest/api/latest/server/resume"
-      curl -u "${ADMIN_USERNAME}:${ADMIN_PASSWORD}" -X POST "${resume_bamboo_url}"
+      local RESULT=$(curl -u "${ADMIN_USERNAME}:${ADMIN_PASSWORD}" -X POST "${resume_bamboo_url}")
+      if [[ "x${RESULT}" == *"RUNNING"* ]]; then
+        SERVER_STATUS="RUNNING"
+      fi
+    fi
+    if [ -z $SERVER_STATUS ]; then
+      log "We were not able to login into the Bamboo software to resume the server." "Warning"
```
Comment (nanux) [inline install.sh:240]:
```suggestion
      log "We were not able to login into the Bamboo software to resume the server." "WARN"
```

File: install.sh [hunk 2]
```diff
@@ -230,7 +231,14 @@ resume_bamboo_server() {
       fi
       bamboo_url=$(terraform output | grep '"bamboo" =' | sed -nE 's/^.*"(.*)".*$/\1/p')
       resume_bamboo_url="${bamboo_url}/rest/api/latest/server/resume"
-      curl -u "${ADMIN_USERNAME}:${ADMIN_PASSWORD}" -X POST "${resume_bamboo_url}"
+      local RESULT=$(curl -u "${ADMIN_USERNAME}:${ADMIN_PASSWORD}" -X POST "${resume_bamboo_url}")
+      if [[ "x${RESULT}" == *"RUNNING"* ]]; then
```
Comment (nanux) [inline install.sh:235]: Should we retry or do you want to rely on the delay?

Comment (nghazali) [reply install.sh:235]: Delay works consistently, but yes, retry is a better approach. I will change this part to retry with a shorter interval and a longer expiry time.

Comment (nghazali) [reply install.sh:235]: However, retry is a better approach in that case. We ca change that part to retry with a shorter interval and a longer expiry time in a new ticket.

**Review by yzha645 — APPROVED** [review]

**Review by nanux — APPROVED** [review]

**Pull request merged**